app/controllers/admin/rooms_controller.rb

```diff
@@ -0,0 +1,49 @@
+class Admin::RoomsController < AdminController
+  before_action :load_room_by_id, only: %i(destroy show)
+  def index
+    @room = Room.new
+    @rooms = Room.all.page(params[:page]).per Settings.rooms.room_per_page
+
+    @picture = @room.pictures.build
+
+    @type_arr = Type.all.map{|type| [type.name, type.id]}
+    @view_arr = View.all.map{|view| [view.name, view.id]}
```

Comment:
dùng pluck đi, so sánh pluck vs select

Reply:
- Select sẽ dùng khi ta cần đầu ra là ActiveRecord model hay dùng để chain truy vấn, trong khi pluck sẽ trả về thẳng cho chúng ta số liệu để phục vụ cho mục đích tính toán.
- Ở đây em chỉ cần một mảng bao gồm name và id thì nên sử dụng pluck => thời gian truy vấn nhanh hơn select